internal/provider/resource_docker_network.go

```diff
@@ -96,6 +96,13 @@ func resourceDockerNetwork() *schema.Resource {
 				Optional:    true,
 				ForceNew:    true,
 			},
+			"ipam_options": {
+				Type:        schema.TypeMap,
+				Description: "Only available with bridge networks. See [bridge options docs](https://docs.docker.com/engine/reference/commandline/network_create/#bridge-driver-options) for more details.",
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
```

Comment:
Ah, this should be `false` (or just remove it completely)

Reply:
`ForceNew` and `Computed`? The snippet's not very clear. I am going to revise the text though to match what I put in the docs.

Reply:
The docs are auto-generated, so you only have to modify the text here.
And simply remove the `Computed` property
```suggestion
```